pkg/cmd/namespace/namespace_test.go

```diff
@@ -1,16 +1,16 @@
-package namespace_test
+package namespace
 
 import (
+	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
 	"testing"
 
 	"github.com/google/go-cmp/cmp"
-	"github.com/jenkins-x-plugins/jx-gitops/pkg/cmd/namespace"
 	"github.com/jenkins-x/jx-helpers/v3/pkg/files"
-	"github.com/jenkins-x/jx-helpers/v3/pkg/kyamls"
+	"github.com/keskad/jx-helpers/v3/pkg/kyamls" // todo: revert before merging
```

Comment:
Before merging please merge https://github.com/jenkins-x/jx-helpers/pull/362 , then switch there back to JX repository and bump a version.